src/reconstruction/StandardModel.jl

```diff
@@ -1,3 +1,4 @@
+using Base: upperbound, lowerbound
```

Comment:
no namespace imports outside src/COBREXA.jl (this may sneakily break other code)

Reply:
also I guess that typing Base.upperbound may be sufficient here, right?

Reply:
I am not even sure why I was doing this... deleting